orthos2.spec

```diff
@@ -128,11 +128,8 @@ ln -sf service %{buildroot}%{_sbindir}/rcorthos2_taskmanager
 ln -sf service %{buildroot}%{_sbindir}/rcorthos2
 ln -sf service %{buildroot}%{_sbindir}/rcorthos2_debug
 %endif
-# This should go into setup.py - but copying tons of non *.py files recursively
-# is cumbersome...
+# This should go into setup.py - but copying tons of non *.py files recursively is cumbersome...
 cp -r orthos2/frontend/static /%{buildroot}/%{python3_sitelib}/orthos2/frontend
-# ToDo: Try to separate the html templates somewhere else
-cp -r templates/* %{buildroot}/%{python3_sitelib}/orthos2
```

Comment:
Update: I guess we want to keep this and you add a quirk to your local test env as well.
In another try/step this is ideally fixed at the root: Use pip installer/wheel instead of setuptools and also copy html/template files via python code (and remove this one then).